cmd/storagenode/alpha/dashboard.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+./storagenode dashboard --config-dir /app/config --identity-dir /app/identity
```

Comment:
For now we should also have this argument: `--external-address=${ADDRESS}`. It is necessary for the storage node to request the bootstrap node to ping it with the proper external address.

Reply:
Completely forgot about this! I'll add it in thanks @kaloyan-raev !